.github/workflows/regression_tests.yml

```diff
@@ -93,7 +93,15 @@ jobs:
           environment: ${{ inputs.environment }}
           pull_request_id: ${{ inputs.pull_request_id }}
           tags: ${{ inputs.tags }}
+          client_id: ${{ secrets.CLIENT_ID }}
+          client_secret: ${{ secrets.CLIENT_SECRET }}
+          certificate: ${{ secrets.CERTIFICATE }}
+          private_key: ${{ secrets.PRIVATE_KEY }}
         run: |
+            export CLIENT_ID=$client_id
```

Comment:
Can these exports just be upper-cased above?

Reply:
just to clarify, would you preferred upper-cased within the `env:` ?

Reply:
That's right, yeah. Those should then be available in the os.getenvs in the test code.